src/features/movie/components/cast/Cast.tsx

```diff
@@ -0,0 +1,47 @@
+import { useState } from "react";
+
+import { MovieProps } from "models/movie.model";
+
+import { FullCastAndCrew } from "./fullCast&Crew/FullCastAndCrew";
+import { TopBilledCast } from "./topBilledCast/TopBilledCast";
+
+import style from "./cast.module.scss";
+
+export const Cast: React.FC<MovieProps> = ({ movie }) => {
+  const [active, setActive] = useState<boolean>(true);
+  const { top_billed_cast, full_cast_and_crew } = movie;
+
+  return (
+    <div className={style.cast}>
+      <div className={style.cast_menu}>
+        <button
+          className={active ? style.active : ""}
+          onClick={(e: React.MouseEvent<HTMLButtonElement, MouseEvent>) =>
+            setActive(true)
+          }
+        >
+          Top Billed Cast
+        </button>
+        <button
+          className={active ? "" : style.active}
```

Comment:
This looks weird, I'm reading it as "if it's active apply no class, otherwise, apply active". I'd expect the inverse. Can you check if this is correct, and if so rewrite it for readability?

Reply:
It mekes sense, I will do it better.